onnxruntime/core/mlas/lib/mlasi.h

```diff
@@ -1297,6 +1298,8 @@ struct MLAS_PLATFORM {
     // TODO: move to cpuinfo
     bool Avx2Supported_ = false;
     bool Avx512Supported_ = false;
+    bool ArmNeonQuantAUnsigned = false;
```

Comment:
can this name be more specific? without context, "A" is not very meaningful.

Reply:
Generally in the context of Matmul development, I think "A" is reasonably well-known to be the first input - but I can go ahead and name it `ArmNeonQuantActivationsUnsigned ` to re-inforce that it is indicating activations (first input). Would that be reasonable ?